src/main/java/frc/robot/shuffleboard/ShuffleboardWidgets.java

```diff
@@ -63,197 +84,233 @@
-      .withPosition(2, 1)
-      .withSize(2, 1)
-      .getEntry();
-      tab.add("flywheelTemperature2", flywheelTemperature[1] * (9 / 5) + 32)
-      .withPosition(4, 1)
-      .withSize(2, 1)
-      .getEntry();
-      tab.add("hoodDegrees",hoodDegrees)
-      .withWidget(BuiltInWidgets.kDial)
-      .withProperties(Map.of("min", 0, "max", 360))
-      .withPosition(8, 3)
-      .withSize(2, 2)
-      .getEntry();
-      tab.add("hoodVelocity",hoodVelocity)
-      .withPosition(6, 1)
-      .withSize(2, 1)
-      .getEntry();
-
-      leftEncoderEntry = tab
-      .add("cellCount",cellCount)
-      .withPosition(8, 1)
-      .withSize(2, 1)
-      .getEntry();
-
-      tab.add("isBrakeset",isBrakeset)
-      .withWidget(BuiltInWidgets.kBooleanBox)
-      .withPosition(0, 2)
-      .withSize(2, 1)
-      .getEntry();
-      tab.add("isLiftFullyExtended",isLiftFullyExtended)
-      .withWidget(BuiltInWidgets.kBooleanBox)
-      .withPosition(2, 2)
-      .withSize(2, 1)
-      .getEntry();
-      tab.add("isLiftFullyRetracted",isLiftFullyRetracted)
-      .withWidget(BuiltInWidgets.kBooleanBox)
-      .withPosition(4, 2)
-      .withSize(2, 1)
-      .getEntry();
-      tab.add("liftExtension",liftExtension)
-      .withPosition(6, 2)
-      .withSize(2, 1)
-      .getEntry();
-      tab.add("isPinned",isPinned)
-      .withWidget(BuiltInWidgets.kBooleanBox)
-      .withPosition(8, 2)
-      .withSize(2, 1)
-      .getEntry();
-
-      tab.add("isWinchEngaged",isWinchEngaged)
-      .withWidget(BuiltInWidgets.kBooleanBox)
-      .withPosition(0, 3)
-      .withSize(2, 1)
-      .getEntry();
+    // Drivetrain Data INCOMPLETE
+    leftEncoderValue = drivetrain.getLeftEncoder();
+    rightEncoderValue = drivetrain.getRightEncoder();
+    // Gyro Datas COMPLETE
+    gyroAngleDegrees = drivetrain.getAngleDegrees();
+    Pose2d pose = drivetrain.getRobotPose();
+    robotX = pose.getTranslation().getX();
+    robotY = pose.getTranslation().getY();
+    robotRotation = pose.getRotation().getDegrees();
+
+    // Turret Data COMPLETE
+    turretDegrees = turret.getPosition() * (1 / Math.PI) * 180;
+    turretVelocity = turret.getVelocity() * (1 / Math.PI) * 180;
+
+    // Shooter Data COMPLETE
+    flywheelVelocity = shooter.getFlywheelSpeed() * (1 / Math.PI) * 180;
+    flywheelTemperature = shooter.getInternalTemperature();
+    hoodDegrees = shooter.getHoodAngle() * (1 / Math.PI) * 180;
+    hoodVelocity = shooter.getHoodVelocity() * (1 / Math.PI) * 180;
+
+    // Magazine Data COMPLETE
+    
+    //cellCount = magazine.getCellCount();
+
+    // Climbing Data COMPLETE
+    isBrakeset = lift.isBrakeSet();
+    isLiftFullyExtended = lift.isLiftFullyExtended();
+    isLiftFullyRetracted = lift.isLiftFullyRetracted();
+    liftExtension = lift.liftExtension();
+    isPinned = lift.isPinned();
+
+    isWinchEngaged = winch.isWinchEngaged();
+
+    // Wheel of Fortune Data MISSING
+
+  }
+
+  private void ShuffleboardView() {
```

Comment:
consider a method name that more accurately communicates purpose of the method...e.g. ShuffleboardInit() would be an appropriate name for this method.